tests/CoreFX/build-and-run-test.cmd

```diff
@@ -0,0 +1,44 @@
+:: Test build and execution wrapper for CoreFX tests
+::
+:: This wrapper is called for each of CoreFX's tests by runtest.cmd
+::
+:: %1 contains test folder
+:: %2 contains test exe name
+::
+@echo OFF
+setlocal ENABLEDELAYEDEXPANSION
+
+set TestFolder=%1
+
+::
+:: We're only interested in referencing the xunit runner - the test dlls will be imported by the test wrapper project
+::
+set TestExecutable=xunit.console.netcore
+set TestFileName=%2
+
+
+:: Copy the artefacts we need to compile and run the xunit exe
+copy /Y "%~dp0\runtest\CoreFXTestHarness\*" "%TestFolder%" >nul
+
+:: Create log dir if it doesn't exist
+if not exist %XunitLogDir% md %XunitLogDir%
+
+if not exist %TestFolder%\%TestExecutable%.exe ( 
+    :: Not a test we support yet, exit silently
```

Comment:
Tests not run by Helix (e.g. AssemblyResolve). Technically, this shouldn't ever happen, but in case the test format changes, this degrades more gracefully.
Ideally, the need for this will be completely eliminated once we add a CoreRT parallel to the uap-aot target for CoreFX.